configure.ac

```diff
@@ -3165,12 +3165,48 @@ if test "$have_glibc_memmove_bug" = yes; then
     AS_VAR_APPEND([LIBMPDEC_CFLAGS], [" -U_FORTIFY_SOURCE"])
 fi
 
+dnl Check for SQLite library. Use pkg-config if available.
+AC_SUBST([PY_SQLITE_CFLAGS])
+AC_SUBST([PY_SQLITE_LDFLAGS])
+if test ! -z "$PKG_CONFIG"; then
+  AS_VAR_APPEND([PY_SQLITE_CFLAGS], ["`"$PKG_CONFIG" sqlite3 --cflags-only-I 2>/dev/null`"])
+  AS_VAR_APPEND([PY_SQLITE_LDFLAGS], ["`"$PKG_CONFIG" sqlite3 --libs-only-L 2>/dev/null`"])
```

Comment:
I recommend to use full cflags and ldflags here. I also have some code that handles other flags.

Reply:
SGTM